WalletWasabi.Fluent/ViewModels/Wallets/Receive/AddressViewModel.cs

```diff
@@ -1,32 +1,42 @@
 ﻿using System;
+using System.Reactive;
 using System.Threading.Tasks;
 using System.Windows.Input;
 using Avalonia;
 using NBitcoin;
 using ReactiveUI;
+using WalletWasabi.Blockchain.Analysis.Clustering;
 using WalletWasabi.Blockchain.Keys;
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets.Receive
 {
 	public partial class AddressViewModel : ViewModelBase
 	{
-		[AutoNotify] private string _label;
+		[AutoNotify] private SmartLabel _label;
 		[AutoNotify] private string _address;
 
-		public AddressViewModel(HdPubKey model, Network network, Func<HdPubKey, string, Task> hideCommand)
+		public AddressViewModel(ReceiveAddressesViewModel parent, HdPubKey model, Network network, Func<HdPubKey, string, Task> hideCommand)
 		{
 			Model = model;
 			_address = model.GetP2wpkhAddress(network).ToString();
 			_label = model.Label;
 
 			CopyAddressCommand = ReactiveCommand.CreateFromTask(async () => await Application.Current.Clipboard.SetTextAsync(Address));
 			HideAddressCommand = ReactiveCommand.CreateFromTask(async () => await hideCommand.Invoke(model, Address));
```

Comment:
Now we have the parent here so `parent.HideAddressAsync(model, Address)` can be used here and remove the `hideCommand` parameter completely.

Just make the `HideAddressAsync()` to public.